_includes/old_sidebar.html

```diff
@@ -3,13 +3,13 @@
 <div class="card" style="margin-bottom: 15px;">
   <div class="card-header">Latest releases</div>
   <div class="card-body">
-         <strong>QuTiP 5.1.1</strong>:
+         <strong>QuTiP 5.2.0</strong>:
          <ul>
-           <li>January 15, 2025</li>
-           <li><a href="https://qutip.readthedocs.io/en/qutip-5.1.x/installation.html">install</a></li>
-           <li><a href="https://qutip.readthedocs.io/en/qutip-5.1.x/">docs</a></li>
-           <li><a href="https://github.com/qutip/qutip/releases/tag/v5.1.1">src</a></li>
-           <li><a href="https://qutip.readthedocs.io/en/qutip-5.1.x/changelog.html">changelog</a></li>
+           <li>June 6, 2025</li>
+           <li><a href="https://qutip.readthedocs.io/en/v5.2.0/installation.html">install</a></li>
+           <li><a href="https://qutip.readthedocs.io/en/v5.2.0/">docs</a></li>
+           <li><a href="https://github.com/qutip/qutip/releases/tag/v5.2.0">src</a></li>
+           <li><a href="https://qutip.readthedocs.io/en/v5.2.0/changelog.html">changelog</a></li>
```

Comment:
```suggestion
           <li><a href="https://qutip.readthedocs.io/en/qutip-5.2.x/installation.html">install</a></li>
           <li><a href="https://qutip.readthedocs.io/en/qutip-5.2.x/">docs</a></li>
           <li><a href="https://github.com/qutip/qutip/releases/tag/v5.2.0">src</a></li>
           <li><a href="https://qutip.readthedocs.io/en/qutip-5.2.x/changelog.html">changelog</a></li>
```
The links intentionally pointed to the release branch rather than the release tag, because that allows users to view the documentation updates immediately.